app/controllers/user_mentee_applications_controller.rb

```diff
@@ -3,7 +3,7 @@ class UserMenteeApplicationsController < ApplicationController
   before_action :set_active_cohort, only: %i[index new create]
 
   def index
-    authorize :user_only_policy, :applicant?
+    authorize :user_only, :applicant?
```

Comment:
I had to change this because I made a mistake when I made the recommendation to @afogel in #326 

Policies are looked up without the `_policy` part attached. So it needs to just be `:user_only`.